src/utils.ts

```diff
@@ -136,8 +136,20 @@ export function getArtifactPath(sourcePath: string, paths: ProjectPathsConfig):
     return path.join(paths.starknetArtifacts, suffix);
 }
 
+/**
+ * Adapts path relative to the root of the project
```

Comment:
Mention that tilde is resolved to homedir

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/317/commits/f8edb2c0f52589ea5e309a3e67e6b6a289fb6a5c